dhis-2/dhis-web/dhis-web-dashboard-integration/src/main/webapp/dhis-web-dashboard-integration/dashboard.vm

```diff
@@ -103,6 +103,11 @@ jQuery( document ).ready( function() {
     <input type="button" class="interpretationButton" value="$i18n.getString( 'share' )" onclick="dhis2.db.shareInterpretation()">
 </div>
 
+<div id="shareFormError">
+    <p>$i18n.getString( 'cannot_write_interpretation' )</p>
+    <input type="button" class="interpretationButton" value="$i18n.getString( 'close' )" onclick="$(this).closest('.ui-dialog-content').dialog('close');">

```

Comment:
To be coherent with the look & feel. Can we use something like
`<a class="greyButtonLink" href="javascript:dhis2.db.addDashboard()" style="margin-top: 1px">$i18n.getString( "create" )</a>` ?
Extracted from a few lines below..